WcaOnRails/db/migrate/20170316030443_add_big_json_to_competitions.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class AddBigJsonToCompetitions < ActiveRecord::Migration[5.0]
+  def change
+    add_column :Competitions, :big_json, :text
```

Comment:
That was actually exactly how I interpreted your question =) This is entirely separate data coming from the outside.

Big picture stuff: I imagine that as the big json spec settles and the WCA website grows more features, more of the big json will be generated rather than read from the database field. @viroulep will be doing this very soon with registration data, which we already have tables for, but do not yet include in the big_json export.